sdk/features/main.py

```diff
@@ -1,37 +1,10 @@
-import argparse
-import os
 import sys
 
-from features.feature_extractor import extract_all_features_from_package
-from features.feature_file_writer import write_features_to_file
-from features.get_project_root import get_project_root
+from lusidfeature.entrypoint import extract_features_to_file
```

Comment:
having seen it now, we should change this to `lusidfeature.reporter` 

sorry!

Reply:
updated!